packages/rum/src/domain/record/serialize.ts

```diff
@@ -167,6 +169,7 @@ export function serializeElementNode(element: Element, options: SerializeOptions
       },
       childNodes: [],
       isSVG,
+      isShadowHost,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏For hidden nodes, `isShadowHost` should not matter. I would remove this property and move the `const isShadowHost = ...` closer to where it is used